src/igvfd/types/file_set.py

```diff
@@ -86,6 +86,34 @@ class MeasurementSet(FileSet):
     item_type = 'measurement_set'
     schema = load_schema('igvfd:schemas/measurement_set.json')
 
+    @calculated_property(
+        condition='multiome_size',
+        schema={
+            'title': 'Related Multiome Datasets',
+            'description': 'Related datasets included in the multiome experiment this measurement set is a part of.',
+            'type': 'array',
+            'uniqueItems': True,
+            'items': {
+                'title': 'Related Multiome Dataset',
+                'description': 'Related dataset included in the multiome experiment this measurement set is a part of.',
+                'type': 'string',
+                'linkTo': 'MeasurementSet'
+            },
+            'notSubmittable': True,
+        }
+    )
+    def related_multiome_datasets(self, request, accession, samples=None):
+        related_datasets = []
+        if samples:
+            for sample in samples:
+                sample_object = request.embed(sample, '@@object')
+                for file_set in sample_object['file_sets']:
+                    if 'measurement-sets' in file_set and \
```

Comment:
are file_sets embedded in the sample?

Reply:
No, just a list of links. i.e. [/measurement-sets/IGVFDS0000AAAA/,/analysis-sets/IGVFDS1111BBBB/]